install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -760,12 +760,12 @@ data:
 {{- end }}
 
 {{- if hasKey .Values "hostPort" }}
-{{- if eq $kubeProxyReplacement "partial" }}
+{{- if or (eq $kubeProxyReplacement "partial") (eq $kubeProxyReplacement "false") }}
```

Comment:
KPR partial was removed in recent releases, so I think we just need the == false check